cmake/onnxruntime_providers.cmake

```diff
@@ -501,6 +501,9 @@ if (onnxruntime_USE_TENSORRT)
   set(CXX_VERSION_DEFINED TRUE)
   add_subdirectory(${ONNXRUNTIME_ROOT}/../cmake/external/onnx-tensorrt)
   set(CMAKE_CXX_FLAGS ${OLD_CMAKE_CXX_FLAGS})
+  if ( CMAKE_COMPILER_IS_GNUCC )##slx
+    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-unused-parameter")
```

Comment:
provide more detail about this change?
is it supposed to change the behavior in line 499?
is line 499 still needed?

Reply:
In TRT7 only the parser needs the two -Wno in line 499, while since TRT8, TRT itself also needs -Wno-unused-parameter.

Reply:
ok. thanks. I missed that the flags are reset in 505.